client/gefyra/api/install.py

```diff
@@ -51,7 +51,10 @@ def install(
     from gefyra.configuration import ClientConfiguration
 
     config = ClientConfiguration(
-        kube_config_file=kubeconfig, kube_context=kubecontext, ignore_docker=True
+        kube_config_file=kubeconfig,
+        kube_context=kubecontext,
+        ignore_docker=True,
+        registry_url=kwargs.get("registry_url", ""),
```

Comment:
Why not make it a parameter?

Reply:
Oh sure thing

Reply:
Ah I can't. In line 69 the kwargs are being transfered to GefyraInstallOptions thus removing registry_url from kwargs results in a faulty install config